org.opent2t.onboarding.hue/js/common.js

```diff
@@ -0,0 +1,22 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
Since both wink and nest included this, I included it in hue for consistency. @arjun-msft any comments?